ironfish-cli/src/commands/debug.ts

```diff
@@ -69,6 +72,8 @@ export default class Debug extends IronfishCommand {
       ['Node version', `${process.version}`],
       ['ironfish in PATH', `${cmdInPath.toString()}`],
       ['Telemetry enabled', `${telemetryEnabled}`],
+      ['Node Name', `${nodeName ? nodeName : 'NONE'}`],
+      ['Block Graffiti', `${blockGraffiti ? blockGraffiti : 'NONE'}`],
```

Comment:
not sure we need this check here. `NONE` is a valid node name and graffiti. would just print the string.

nit: lowercase n and g :) just to keep it consistent

Reply:
It's probably fine to just print out the block graffiti and leave it blank.

Reply:
@wd021 I removed this checking, thanks for comment :)